doc/cli_vs_yum.rst

```diff
@@ -70,12 +70,6 @@ The Yum version of this command is maintained for legacy reasons only. The user
 can just do ``dnf provides`` to find out what package gives a particular
 provide.
 
-===========================
- No ``deplist`` command
-===========================
-
```

Comment:
Here I am not sure if it should be deleted. Command ``yum deplist`` will works but not ``dnf deplist``. For dnf you have to use ``dnf repoquery --deplist``. @jsilhan Is that what was supposed to? And how to handle descriptions in cli_vs_yum.rst ?